lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -1735,6 +1744,11 @@ GLStatus TFGraphLowering::visitGraphOperationInst(GraphOperationInst *inst) {
   }
 
   // Process all of the attributes.
+  // For an inst like:
+  //   graph_op "Const"() {dtype$dtype: $Builtin.Int64, value$tensor: i1 0
+  // We will use the `dtype` attr value to lower the `value` attr, so we
+  // remember the dtype info here.
+  unsigned dtype = 0;
```

Comment:
added  a comment:
```
// We use unsigned instead of TF_DataType, to express the invalid initial
// value 0.
```

Good point on the inner dtype. Renamed that local var.